MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -816,7 +822,7 @@ private void SetCollider(ColliderPatch colliderPatch)
 
 			if (colliderType == ColliderType.Auto)
 			{
-				colliderGeometry = App.AssetCache.GetColliderGeometry(MeshId);
+				colliderGeometry = App.AssetManager.GetById(MeshId).Value.ColliderGeometry;
```

Comment:
>.GetById(MeshId).Value [](start = 39, length = 22)

Is this safe to just dereference when the GetBy Id can return null?

Reply:
The `SetCollider` method is never called unless the mesh is loaded. There are guards in `PatchCollider` and `PatchAppearance` handling the loading cases.